.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+install:
+  - docker build -t quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7} .
+
+script:
+  - docker run -d --name rwd quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7}
```

Comment:
When running it locally, it opens a Python REPL, I think because of the base image. I'll try and think of a quick way to make sure things are good.